Example/package.json

```diff
@@ -34,17 +34,16 @@
     "@babel/preset-env": "^7.20.0",
     "@babel/runtime": "^7.12.5",
     "@react-native/eslint-config": "^0.72.1",
-    "@react-native/metro-config": "^0.72.1",
-    "@tsconfig/react-native": "^2.0.2",
+    "@react-native/metro-config": "^0.72.5",
+    "@tsconfig/react-native": "^3.0.0",
     "@types/jest": "^29.2.1",
-    "@types/metro-config": "^0.76.1",
```

Comment:
`"@types/metro-config": "^0.76.3",` is missing

Reply:
Getting the following error with `@types/metro-config` installed, skipping this change as for now.
```
$ /Users/tomekzaw/RNOS/SeventyTwo/react-native-reanimated/FabricExample/node_modules/.bin/tsc --noEmit
error TS2688: Cannot find type definition file for 'metro-config'.
  The file is in the program because:
    Entry point for implicit type library 'metro-config'
```